__tests__/app.test.js

```diff
@@ -107,3 +107,36 @@ describe("* GET/api/articles", () => {
       });
   });
 });
+
+describe("* GET/api/articles/:article_id", () => {
+  test("returns status code 200", () => {
+    return request(app).get("/api/articles/3").expect(200);
+  });
+
+  test("responds with a single matching article", () => {
+    return request(app).get("/api/articles/3").expect(200)
+    .then(({ body }) => {
+      expect(body).toEqual(
+        {
+          article_id: 3,
+          title: 'Eight pug gifs that remind me of mitch',
+            topic: 'mitch',
+            author: 'icellusedkars',
+            body: 'some gifs',
+            created_at: '2020-11-03T09:12:00.000Z',
+            votes: 0
+        }
```

Comment:
We want to make this test less "brittle" so the article is open to extension (e.g. if another property is added to the article data in future). To do this we need to move away from `toEqual` and rely on flexible matches like `toMatchObject` or `expect.objectContaining` which will match for properties but won't fail if more properties are present in future.

I'd refactor this test to be more flexible in that sense

Reply:
this has been refactored to match previous tests. thank you